keys/generate-keys-and-certs.sh

```diff
@@ -40,3 +46,11 @@ echo "[INFO]: rootcert.fingerprint: The root CA certificate's fingerprint"
 echo "[INFO]: signing-key-N.cert:   The certificate corresponding to key N"
 echo "[INFO]: signing-key-N.key:    Key N's private key"
 
+# SSH key generation for cpu
+
+echo "[INFO]: Generating ssh keys for using the cpu command"
+
+ssh-keygen -b 2048 -t rsa -f "${cpu_key_dir}/ssh_host_rsa_key" -q -N ""
+ssh-keygen -b 2048 -t rsa -f "${cpu_key_dir}/cpu_rsa" -q -N ""
+
+echo "[INFO]: Key generation for CPU command successfull!"
```

Comment:
Is it correct, that _cpu_rsa_ is a key pair explicitly needed by the _cpu server_ - _cpu client_ connection, whereas the _ssh_host_rsa_ key pair could also be replaced by your usual ssh key pair?

Reply:
It's the other way around

Reply:
ok ^^ maybe you can find alternative names for the two keypairs or doument what it what